crates/ruma-events/src/room/avatar.rs

```diff
@@ -50,7 +50,7 @@ impl RoomAvatarEventContent {
 
     /// Create an empty `RoomAvatarEventContent`.
     ///
-    /// With the `unstable-pre-spec` feature, this method takes an `MxcUri`.
+    /// Without the `unstable-pre-spec` feature, this method takes an `MxcUri`.
```

Comment:
Since `docs.rs` doesn't use `unstable-pre-spec` (I'm pretty sure), I think we should keep the "With the `unstable-pre-spec`" wording and change the latter part of the sentence.

Reply:
There's also the docs without the feature above.

Reply:
I also checked, `docs.rs` uses the feature.

Reply:
What a mess 😞

I've had a look and the "Without" wording is definitely less common, but if we use `unstable-pre-spec` for docs.rs then that's not great either. Hopefully we can just get rid of most of these non-additive unstable features over the course of this year..